CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+# Releasing 0.5.1 (unreleased)
+
+  * Drop support for Ruby 1.8.7
+  * voxpupuli namespace release
```

Comment:
Now the README is up-to-date, so technically this works. You reminded me to add our build flair to the README though.